Gruntfile.js

```diff
@@ -146,6 +146,88 @@ module.exports = function( grunt ) {
 				"tween"
 			]
 		},
+		karma: {
+			options: {
+				customContextFile: "test/karma.context.html",
+				customDebugFile: "test/karma.debug.html",
+				frameworks: [ "qunit" ],
+				middleware: [ "mockserver" ],
+				plugins: [
+					"karma-*",
+					{
+						"middleware:mockserver": [
+							"factory",
+							require( "./test/middleware-mockserver.js" )
+						]
+					}
+				],
+				files: [
+					"test/data/jquery-1.9.1.js",
+					"external/qunit-assert-step/qunit-assert-step.js",
+					"external/sinon/sinon.js",
+					"external/npo/npo.js",
+					"external/requirejs/require.js",
+					"test/data/testinit.js",
+
+					"dist/jquery.min.js",
+
+					// Replacement for testinit.js#loadTests()
+					"test/data/testrunner.js",
+					"test/unit/basic.js",
+					"test/unit/core.js",
+					"test/unit/callbacks.js",
+					"test/unit/deferred.js",
+					"test/unit/deprecated.js",
+					"test/unit/support.js",
+					"test/unit/data.js",
+					"test/unit/queue.js",
+					"test/unit/attributes.js",
+					"test/unit/event.js",
+					"test/unit/selector.js",
+					"test/unit/traversing.js",
+					"test/unit/manipulation.js",
+					"test/unit/wrap.js",
+					"test/unit/css.js",
+					"test/unit/serialize.js",
+					"test/unit/ajax.js",
+					"test/unit/effects.js",
+					"test/unit/offset.js",
+					"test/unit/dimensions.js",
+					"test/unit/animation.js",
+					"test/unit/tween.js",
+					"test/unit/ready.js",
+
+					{ pattern: "dist/jquery.js", included: false, served: true },
```

Comment:
That'll be a future improvement, but the way it would work is basically you create another karma task, one like the current one that includes `dist/jquery.min.js` in `files` and one that includes `dist/jquery.js` instead.

I had to whiltelist the file even in the current Grunt karma task because one of the iframes we load in the tests load `test/jquery.js`, which then loads `dist/jquery.js` in some cases.